datasets/cn/national_congress/cn_national_congress.yaml

```
@@ -0,0 +1,44 @@
+title: China National People's Congress
+entry_point: crawler.py
+prefix: cn-npc
+coverage:
+  frequency: never 
+  start: 2024-05-16
+  end: 2028-03-01
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Current members of the Chinese National People's Congress
+description: |
+  The National People's Congress (NPC) is the highest organ of
+  state power of the People's Republic of China. It is the only 
+  branch of the Chinese government, and is unicameral. The NPC
+  is elected for a term of five years, and the last election
+  was held in 2023. Delegates to the National People's Congress
+  are allowed to hold seats in other bodies of government
+  simultaneously and the party and the NPC typically includes
+  all of the senior officials in Chinese politics.
+publisher:
+  name: 全国人民代表大会
+  description: National People's Congress
+  url: http://www.npc.gov.cn
+  official: true
+  country: "cn"
+url: http://www.npc.gov.cn
+data:
+  format: CSV
```

Could you paste the data into could you paste the data into https://docs.google.com/spreadsheets/d/1WfvKmydpsyIQe-5V28meRx2miVNCxMAtbtM-taJDFNw/edit#gid=0  then download context.data_url in the crawler. Then we can easily update the data.

```suggestion
data:
  url: https://docs.google.com/spreadsheets/d/e/2PACX-1vRWjzoZCQ4SSJGimuIP_ESCS23ZO2HWO7ENuwNr_4yjiXigeCezqtVz5rUOmUj3AfHeLyAalUiBmG1D/pub?output=csv
  format: CSV
```

---

Do you have any example on how we download the data from the google sheets using python? I tried using a `requests.get` but it didn't work

---

did you see the URL in the code suggestion and the comment are different? The URL in the comment is the sheet editor, while the url in the suggestion serves a CSV file.

What we usually do is "publish to web" from the file menu and choose the csv option. If we need multiple sheets, we create a URL for each worksheet or alter the querystring based on the selected sheet. We then fetch the csv using context.fetch_resource as usual https://github.com/opensanctions/opensanctions/blob/main/datasets/_wikidata/oligarchs/crawler.py#L33